lib/skywalking/tracing_context_test.lua

```diff
@@ -30,24 +31,24 @@ TestTracingContext = {}
     end
 
     function TestTracingContext:testInternal_NextSpanSeqID()
-        local context = TC:new(1, 1)
+        local context = TC.new(1, 1)
 
-        lu.assertEquals(context.internal:nextSpanID(), 0)
+        lu.assertEquals(TC_Internal.nextSpanID(context.internal), 0)
     end
 
     function TestTracingContext:testInternal_addActive()
-        local context = TC:new(1, 1)
+        local context = TC.new(1, 1)
 
         local mockSpan = {span_id = 0}
-        context.internal:addActive(mockSpan)
+        TC_Internal.addActive(context.internal, mockSpan)
 
-        lu.assertEquals(#(context.internal.active_spans), 1)
+        lu.assertEquals(#context.internal.active_spans, 1)
     end
 
     function TestTracingContext:testSpanStack()
-        local context = TC:new(1, 1)
-        local span1 = context:createEntrySpan('entry_op')
-        local span2 = context:createExitSpan("exit_op", span1, "127.0.0.1")
+        local context = TC.new(1, 1)
+        local span1 = TC.createEntrySpan(context, 'entry_op')
+        local span2 = TC.createExitSpan(context, "exit_op", span1, "127.0.0.1")
```

Comment:
Why use this code style?  In the tracing concept, there should be one context, which hosts all inform to connect spans.
I put the codes in the old way in order to keep the internal context in the shadow, and open limited APIs.

Reply:
got it. I will hide the internal context.
There are two reasons for this code style: 
- In nginx lua, top level variables are shared by all requests in a worker, so top level variables are easy to race condition, so it needs to be read-only
- There is no class in Lua, which needs to be simulated by table. Therefore, in nginx Lua, object-oriented programming mode is generally not recommended

Reply:
> There is no class in Lua, which needs to be simulated by table. Therefore, in nginx Lua, object-oriented programming mode is generally not recommended

I am aware of this, but if refactoring all codes to out of OO, it will be hard to across check the the first-class java agent.